cmd/nerdctl/version.go

```diff
@@ -66,7 +67,17 @@ func versionAction(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	v, vErr := versionInfo(cmd, globalOptions)
+	address := globalOptions.Address
+	// rootless `nerdctl version` runs in the host namespaces, so the address is different
```

Comment:
```suggestion
	// rootless `nerdctl version` runs in the host namespaces (parent), so the address is different
```

Reply:
"host namespaces" should be already enough